src/v3/src/hooks/usePolling.ts

```diff
@@ -100,6 +107,19 @@ export const usePolling = (
         stateHandle: stateToken && idxTransaction?.context?.stateHandle,
         ...payload,
       });
+
+      // @ts-expect-error OKTA-585869 errorCode & errorIntent properties missing from context type
+      if (newTransaction?.context?.errorCode === 'E0000047' && !newTransaction?.context?.errorIntent) {
+        // When polling encounter rate limit error, wait 60 sec for rate limit bucket to reset before polling again
+        const clonedTransaction = cloneDeep(idxTransaction);
```

Comment:
cloned the transaction to avoid overriding the `refresh` property from original transaction.